README.md

```diff
@@ -218,21 +218,29 @@ not enabled) as reported by the `cpuid` instruction.
 | hardware_multithreading | <br>           | Hardware multithreading, such as Intel HTT, enabled (number of locical CPUs is greater than physical CPUs)
 | power                   | sst_bf.enabled | Intel SST-BF ([Intel Speed Select Technology][intel-sst] - Base frequency) enabled
 
-### X86 CPUID Features (Partial List)
+### CPUID Features
+The (sub-)set of CPUID features to publish is configurable via the
+`attributeBlacklist` and `attributeWhitelist` options of the cpuid source. If
+whitelist is specified, only whitelisted attributes will be published. With
+blacklist, only blacklisted attributes are filtered out. `attributeWhitelist`
+has priority over `attributeBlacklist`.
+For examples and more information about configurability, see
+[Configuration Options](#configuration-options).
+
+By default, the following CPUID flags have been blacklisted:
+BMI1, BMI2, CLMUL, CMOV, CX16, ERMS, F16C, HTT, LZCNT, MMX, MMXEXT, NX, POPCNT,
+RDRAND, RDSEED, RDTSCP, SGX, SSE, SSE2, SSE3, SSE4.1, SSE4.2 and SSSE3.
```

Comment:
@mythi what do you think about by default blacklisted SGX?

Reply:
SGX cpuid label is pretty worthless, if not even misleading as is discussed in #130

Reply:
+1, disabling it makes sense for now